### ISSUE TITLE
[TODO] Database

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/radio-logger#33]
This application will use several databases and tables

#1) database for storing the records of at least one published periodical/magazine and publication.. particularly one called "Passport to World Band Radio", which had listings published in order by Frequency during the late 80s to the early 2000's. 

#2)  will store the logbook records of the radio contacts and their calls made by amateur radio operators,

#3)  will store the logbook records of Shortwave radio Listeners primarily for QSL purposes.

#4)  will house user records, login, etc.

**Comment by gbowne1** [comment]
There is a Heroku PostgreSQL connection available, however it's attached to the other repository.  I can document this if necessary to the contributors.  Can the GitHub repository link be changed in Heroku so it links to this repository instead of the other?

Some stuff needs to happen in possibly several milestones.

-Determine SQL needs for all the related databases
  - Users
  - Data
  - Login
  - other needs
 
When this App gets released in a native format (HTML, JavaScript, CSS, jQuery, etc) we will be using MariaDB/MySQL.

**Comment by Ej1seven** [comment]
Can you please document the information needed to link to your Heroku PostgreSQL database.  I would like to get started creating a login page and setting up user profiles but I need your database information in order to store this information. 

Thank you!

**Comment by gbowne1** [comment]
I figured out how to change the link in the Heroku dashboard.

The Heroku PostgreSQL is here: 
https://dashboard.heroku.com/apps/radiologger

**Comment by Ej1seven** [comment]
I believe you have to provide me access to the database in order for me to connect. I attached a screenshot of what my webpage displays when I attempt to access the database. 

![Screenshot (39)](https://user-images.githubusercontent.com/12377787/150241899-d2c3efde-d905-45c7-8d07-bd04f3ae331b.png)

**Comment by gbowne1** [comment]
yes I have to give you access to it.  I was out today and didn't get a chance to do that till now. 
Do you have a gmail or email address you use?

**Comment by Ej1seven** [comment]
ej1seven@gmail.com

**Comment by gbowne1** [comment]
Ok, Thanks.  I'll add you.

**Comment by gbowne1** [comment]
Added.. should work now.  I don't know a ton about Heroku PostgreSQL.

**Comment by gbowne1** [comment]
@Ej1seven did you get anywhere with this?

**Comment by gbowne1** [comment]
@Ej1seven I moved this project to:

https://github.com/gbowne1/radio_logger and changed to just plain Express & Node app.
